SU-HSDO/stanford_mrc
# Mrcd8 267 events subnav

# READY FOR REVIEW
# Summary
- added Subnav in sidebar on events page.

# Needed By (Date)
- End of sprint

# Urgency
- mild

# Steps to Test
1. Ensure you have latest blt and composer update
2. Checkout branch
3. sync to prod and update database
4. view an events node page
5. ensure a subnav menu on the left and the subnav is set for the "Events" menu.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
